### ISSUE TITLE
Touch on 2.8inch

### ISSUE THREAD

**Issue by borkog** [Xinyuan-LilyGO/LilyGo-T-RGB#74]
Hi,
So I couldn't properly compile examples for 2.8inch T-RGB ... Touch does not get initialised properly because 

(Other than using `bool rslt = panel.begin(LILYGO_T_RGB_2_8_INCHES);`)

LilyGo_RGBPanel.cpp is attempting to use TouchDrvCSTXXX() and when this fails, it just exits ... if you comment out everything before `_touchDrv = new TouchDrvGT911();` it works just fine

so... please see what you can do

**Comment by lewisxhe** [comment]
You purchased without touch?

**Comment by borkog** [comment]
No, it worked out of the box with factory sketch. When I compiled "factory" with my wifi ssid and pass touch didnt work. 
I explained what I think the problem was

**Comment by lewisxhe** [comment]
Are you using Arduino IDE or platformio? I tested it and it's normal

**Comment by borkog** [comment]
Arduino IDE, don't know about esp32 core (how do I find out?)

**Comment by lewisxhe** [comment]
![image](https://github.com/user-attachments/assets/3f7f0fbb-379d-4ed6-86d7-7952308ee786)
You can see the version here. And after the compilation is completed, you will also have the version numbers of all dependent libraries displayed in the compilation log window

**Comment by borkog** [comment]
3.0.7 (I don't see 3.1.0 on the list)

**Comment by lewisxhe** [comment]
Open all logs, and then there will be touch detection logs printed at startup. Please check whether the touch has been detected at startup.
```
Arduino tools -> Core Debug Level:  -> Verbose
```

**Comment by lewisxhe** [comment]
The latest version of SensorLib will be pushed to the Arduino library manager tomorrow, v0.2.5 will solve this problem
